examples/source/interactivity-dynamic-content/Dynamic_Menu_With_Sub_Menus.html

```diff
@@ -0,0 +1,167 @@
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+
+  <style amp-custom>
+    #sideNav {
+      width: 100%;
+      height: 100%;
+      background-color: #fff;
+      max-width: 100vw;
+    }
+
+    .header-primary li a {
+      padding: 8px 8px 8px 32px;
+      text-decoration: none;
+      font-size: 16px;
+      color: #818181;
+      display: block;
+      transition: 0.3s;
+      border-bottom: 1px solid #e6e6e6;
+      font-family: "Open Sans", sans-serif;
+      font-size: 16px;
+      font-weight: 400;
+      line-height: 24px;
+      outline: 0;
+    }
+
+    .main-nav {
+      width: 100%;
+      height: 100%;
+      background-color: #fff;
+      max-width: 100vw;
+    }
+
+    .mobile-menu-header {
+      background-color: blue;
+      height: 45px;
+      text-align: center;
+      line-height: 45px;
+      color: white;
+    }
+
+    .closebtn {
+      top: 0px;
+      right: 15px;
+      outline: 0;
+      position: absolute;
+    }
+
+    .innermenulinks {
+      padding: 8px 8px 8px 32px;
+      text-decoration: none;
+      font-size: 16px;
+      color: #818181;
+      display: block;
+      transition: 0.3s;
+      border-bottom: 1px solid #e6e6e6;
+      font-family: "Open Sans", sans-serif;
+      line-height: 24px;
+      font-weight: 400;
+    }
+
+    .rightarrow:after {
+      content: ">";
+      float: right;
+      padding-right: 10px;
+      font-weight: bolder;
+      font-size: 25px;
+    }
+
+    .leftarrow:before {
+      content: "<";
+      float: left;
+      padding-left: 10px;
+      font-weight: bolder;
+      font-size: 25px;
+    }
+
+  </style>
+</head>
+<!-- ## Implementation -->
+  <!--
+      We use two `amp-sidebar` elements nested within eachother.
+      
+      `amp-list` is utilized to retrieve a formatted JSON describing the menu and submenus.
+
+      `amp-mustache` is used for templating. {{items}} and {{submenu}} check for the presence of the keys in the JSON object.      
+      
+  -->
+<body>
+  
+  <header>
+    <h3>
+      Dynamic Sidebar
+    </h3>
+    <p>
+      This side bar is dynamically constructed from a JSON response, which contains a top level and a sub level
+    </p>
+    <button aria-label="open sidebar" on="tap:sideNav.toggle" tabindex="0">
```

Comment:
I don't think the aria-label is needed here as it's the same as the button text.